docs/dev/table/functions/udfs.md

```diff
@@ -1268,693 +1044,770 @@ class WeightedAvg extends AggregateFunction[JLong, CountAccumulator] {
-    *                    aggregated results
-    * @param out         the collector used to output data
-    */
-  def emitValue(accumulator: ACC, out: Collector[T]): Unit // OPTIONAL
-
-  /**
-    * Called every time when an aggregation result should be materialized. The returned value
-    * could be either an early and incomplete result (periodically emitted as data arrive) or
-    * the final result of the aggregation.
-    *
-    * Different from emitValue, emitUpdateWithRetract is used to emit values that have been updated.
-    * This method outputs data incrementally in retract mode, i.e., once there is an update, we
-    * have to retract old records before sending new updated ones. The emitUpdateWithRetract
-    * method will be used in preference to the emitValue method if both methods are defined in the
-    * table aggregate function, because the method is treated to be more efficient than emitValue
-    * as it can outputvalues incrementally.
-    *
-    * @param accumulator the accumulator which contains the current
-    *                    aggregated results
-    * @param out         the retractable collector used to output data. Use collect method
-    *                    to output(add) records and use retract method to retract(delete)
-    *                    records.
-    */
-  def emitUpdateWithRetract(accumulator: ACC, out: RetractableCollector[T]): Unit // OPTIONAL
- 
-  /**
-    * Collects a record and forwards it. The collector can output retract messages with the retract
-    * method. Note: only use it in `emitRetractValueIncrementally`.
-    */
-  trait RetractableCollector[T] extends Collector[T] {
-    
-    /**
-      * Retract a record.
-      *
-      * @param record The record to retract.
-      */
-    def retract(record: T): Unit
-  }
-}
+/*
+ * Merges a group of accumulator instances into one accumulator instance. This method must be
+ * implemented for unbounded session window grouping aggregates and bounded grouping aggregates.
+ *
+ * param: accumulator the accumulator which will keep the merged aggregate results. It should
+ *                    be noted that the accumulator may contain the previous aggregated
+ *                    results. Therefore user should not replace or clean this instance in the
+ *                    custom merge method.
+ * param: iterable    an java.lang.Iterable pointed to a group of accumulators that will be
+ *                    merged.
+ */
+def merge(accumulator: ACC, iterable: java.lang.Iterable[ACC]): Unit
 {% endhighlight %}
 </div>
+
 </div>
 
+{% top %}
+
+Table Aggregate Functions
+-------------------------
+
+A user-defined table aggregate function (_UDTAGG_) maps scalar values of multiple rows to zero, one,
+or multiple rows. The returned record may consist of one or more fields. If an output row consists of
+only one field, the row can be omitted and a scalar value can be emitted. It will be wrapped into an
+implicit row by the runtime.
```

Comment:
```suggestion
A user-defined table aggregate function (_UDTAGG_) maps scalar values of multiple rows to zero, one,
or multiple rows. The returned records may consist of one or more fields. If an output row consists of
only one field, the row can be omitted and a scalar value can be emitted and it will be implicitly wrapped into an row by the runtime.
```

Reply:
I will replace `and` with `that` and fix the `an`.